iamclient/iamclient.go

```diff
@@ -160,6 +160,11 @@ func (client *Client) GetSystemID() (systemID string) {
 	return client.systemID
 }
 
+// GetCurrentNodeInfo returns info for current node.
```

Comment:
Adding new API should be done in separate commit

Reply:
done